BitmovinAdobeAnalytics/README.md

```diff
@@ -1,3 +1,123 @@
-# BitmovinAdobeMediaAnalytics
+# Adobe Analytics Integration for the Bitmovin Player iOS SDK
+This module allows for the integration of your Abode Experience Media Analytics backend with your Bitmovin Player iOS SDK.
 
-A description of this package.
+## Requirements
+-----------------
+1. Adobe Experience Cloud account
+2. Bitmovin account
+3. Adobe Experience Platform for mobile AEP SDKs is configured and "launch-app-id" setup as provided in link [here](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration)
+
+## Getting started
+------------------
+
+## Installation & Configuration
+
+1. BitmovinAdobeAnalytics is available through [CocoaPods](https://cocoapods.org). We depend on cocoapods version >= 1.4.
+
+- To install it, simply add the following line to your Podfile:
+
+```ruby
+pod 'BitmovinAdobeAnalytics', git: 'https://github.com/bitmovin/bitmovin-player-ios-analytics-adobe.git', tag: '1.0.0-rc1'
```

Comment:
```suggestion
pod 'BitmovinAdobeAnalytics', git: 'https://github.com/bitmovin/bitmovin-player-ios-analytics-adobe.git', tag: '1.0.0'
```

Reply:
Keeping this as `1.0.0-rc1` untill 1st integration is done. Any issues reported in integration phase can be fixed before release `1.0.0`

Reply:
Ah, sorry, I thought 1.0.0 is already released. Sounds good 👍